deploy/helm/charts/Chart.yaml

```diff
@@ -16,4 +16,5 @@ sources:
 - https://github.com/openebs/zfs-localpv
 dependencies:
   - name: crds
-    version: "2.5.0"
+    version: 2.5.1
+    condition: crds.enabled
```

Comment:
do we need to document that this has to be disabled when upgrading?

Reply:
I think it's already present in the doc website. https://openebs.io/docs/main/user-guides/upgrade